frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -21,10 +23,13 @@ interface SectionFilterHeaderProps extends SectionHeaderProps {
 
 export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
   const { noNamespaceFilter = false, noSearch = false, ...headerProps } = props;
+  const location = useLocation();
+  const namespaceInURL = getFilterValueByNameFromURL('namespace', location, true);
```

Comment:
Maybe we should just use the value of the namespace filter in Redux here, as it should be the one source of truth.